ref/general/auth-filter.adoc

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+:seo-title: Using authentication filters
+:seo-description: You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+:page-layout: general-reference
+:page-type: general
+= Using authentication filters
+
+You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+
+In applications where different kinds of protected resources are available to different users, authentication filters can control what types of authentication are available for which users. For example, in a banking application, a customer might authenticate by using social media credentials while an administrator must sign in with SPNEGO.
+
+Alternatively, you might want to secure a web application with the Social Login feature, while an associated microservice application is secured with MicroProfile JSON Web Token feature. In this case, you can configure an authentication filter so that the specified security configuration takes effect only when the conditions in the filter are met.
```

Comment:
Social Login feature 
to
Social Media Login feature

Reply:
fixed- thanks